content/swiftui/concepts/views/terms/form/form.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'Form'
+Description: 'Container that collects controls used for data entry. Like `Settings` on iPhones and Macs.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`Form`** view arranges its elements into a column. It collects data entries and settings. The `Form` view is the main element used inside `Settings` on iPhone or iPad and `System Settings` on other devices.
+
+## Syntax
+
+```pseudo
+var body: some View {
+    Form {
+        Subviews here
+    }
+}
+```
+
+## Example
+
+In the example below, a `Form` is wrapped inside a `NavigationView` that will display 4 fields: 1 `Text`, 1 `TextField` where the user can insert their email, 1 `Button` and one `NavigationLink` that leads to an empty view.
```

Comment:
```suggestion
In the example below, a `Form` is wrapped inside a `NavigationView` that will display 4 fields:
```

I recommend explaining the four fields after the code block. Can we also link out to:

- https://www.codecademy.com/resources/docs/swiftui/views/text
- https://www.codecademy.com/resources/docs/swiftui/views/button